pkg/persistence/migrator.go

```diff
@@ -107,13 +107,19 @@ CREATE TABLE IF NOT EXISTS events (
 	UNIQUE KEY identifier_type_date (source, identifier,device_type,date)
 )`,
 		},
-	},{
+	},{ // This is for the granularity release
```

Comment:
Let's strip this out of migrations and run a manual intervention.

Reply:
okay